### PR TITLE
Refactor Actor and Device Liveviews

### PR DESCRIPTION
**Pull request by bmanifold**
Why:

* The previous Actor and Device Liveviews had used static views and data as a starting point for fleshing out the web UI. This commit builds on that and replaces (most) of the static data with data from the database, as well as updating the static Liveview templates to use components where possible.

### CODE REVIEW

File: elixir/apps/web/lib/web/live/devices_live/show.ex [hunk 1]
```diff
@@ -9,136 +17,66 @@ defmodule Web.DevicesLive.Show do
...
-            </td>
-          </tr>
-          <tr class="border-b border-gray-200 dark:border-gray-700">
-            <th
-              scope="row"
-              class="text-right px-6 py-4 font-medium text-gray-900 whitespace-nowrap bg-gray-50 dark:text-white dark:bg-gray-800"
+      <.vertical_table>
+        <.vertical_table_row>
+          <:label>Identifier</:label>
+          <:value><%= @device.id %></:value>
```
Comment (jamilbk) [inline elixir/apps/web/lib/web/live/devices_live/show.ex]: One minor thing here -- we had thought of displaying a truncated device ID here, hence the hardcoded `BF0F8D31FA89`. But now that I think more about it, that might confusing to the user and add needless complexity. So this LGTM

**Review by jamilbk — APPROVED** [review]
LGTM

File: elixir/apps/web/lib/web/components/table_components.ex [hunk 2]
```diff
@@ -250,19 +255,26 @@ defmodule Web.TableComponents do
       </.vertical_table_row>
   """
 
+  attr :label_width, :string, default: nil
+  attr :value_width, :string, default: nil
```
Comment (AndrewDryga) [inline elixir/apps/web/lib/web/components/table_components.ex]: Maybe it's better to name them like:
```suggestion
  attr :label_class, :string, default: nil
  attr :value_class, :string, default: nil
```

Comment (bmanifold) [reply elixir/apps/web/lib/web/components/table_components.ex]: Yeah, that's fair.  I was trying to guide the user in to was type of value was meant to be put there, but looking at it now it probably is better to just open it up to letting any class value be set.

File: elixir/apps/web/lib/web/live/users/edit.ex [hunk 3]
```diff
@@ -1,65 +1,52 @@
 defmodule Web.Users.Edit do
```
Comment (AndrewDryga) [inline elixir/apps/web/lib/web/live/users/edit.ex]: @bmanifold you need to rename modules and folder name to actors too

**Review by AndrewDryga — APPROVED** [review]
One small thing and LGTM.

File: elixir/apps/web/lib/web/live/users/edit.ex (hunk 3 above)
Comment (bmanifold) [reply elixir/apps/web/lib/web/live/users/edit.ex]: Ah, good catch!

Comment (bmanifold) [reply elixir/apps/web/lib/web/live/users/edit.ex]: This has been updated.

File: elixir/apps/web/lib/web/components/table_components.ex (hunk 2 above)
Comment (bmanifold) [reply elixir/apps/web/lib/web/components/table_components.ex]: This has been updated

**Pull request merged**